property.go

```diff
@@ -126,24 +127,25 @@ func getPropertyName(expr ast.Expr, parser *Parser) (propertyName, error) {
 	return propertyName{}, errors.New("not supported" + fmt.Sprint(expr))
 }
 
-func getArrayPropertyName(astTypeArrayElt ast.Expr, parser *Parser) propertyName {
+func getArrayPropertyName(pkgName string, astTypeArrayElt ast.Expr, parser *Parser) propertyName {
 	switch elt := astTypeArrayElt.(type) {
 	case *ast.StructType, *ast.MapType, *ast.InterfaceType:
 		return propertyName{SchemaType: "array", ArrayType: "object"}
 	case *ast.ArrayType:
 		return propertyName{SchemaType: "array", ArrayType: "array"}
 	case *ast.StarExpr:
-		return getArrayPropertyName(elt.X, parser)
+		return getArrayPropertyName(pkgName, elt.X, parser)
 	case *ast.SelectorExpr:
 		return parseFieldSelectorExpr(elt, parser, newArrayProperty)
 	case *ast.Ident:
-		name := TransToValidSchemeType(elt.Name)
+		name := fmt.Sprintf("%s.%s", pkgName, TransToValidSchemeType(elt.Name))
 		if actualPrimitiveType, isCustomType := parser.CustomPrimitiveTypes[name]; isCustomType {
 			name = actualPrimitiveType
 		}
 		return propertyName{SchemaType: "array", ArrayType: name}
```

Comment:
Need to add package name only when find in CustomPrimitiveTypes, same in default case.
```
name := TransToValidSchemeType(elt.Name)
if actualPrimitiveType, isCustomType := parser.CustomPrimitiveTypes[pkgName + "." + name]; isCustomType {
    name = actualPrimitiveType
}
return propertyName{SchemaType: "array", ArrayType: name}
```

Reply:
Sorry, review again and it is better as follows , I think , in case of redefinition of golang primitive types, for example 'type int32 string'
```
case *ast.Ident:
		name := elt.Name
		if actualPrimitiveType, isCustomType := parser.CustomPrimitiveTypes[pkgName + "." + name]; isCustomType {
			name = actualPrimitiveType
		}else{
			name = TransToValidSchemeType(elt.Name)
		}
		return propertyName{SchemaType: "array", ArrayType: name}
	default:
		name := fmt.Sprintf("%s", astTypeArrayElt)
		if actualPrimitiveType, isCustomType := parser.CustomPrimitiveTypes[pkgName + "." + name]; isCustomType {
			name = actualPrimitiveType
		}else{
			name = TransToValidSchemeType(name)
		}
		return propertyName{SchemaType: "array", ArrayType: name}
```